hs_tools_resource/forms.py

```diff
@@ -528,7 +527,7 @@ class Meta:
 
 class ToolIconValidationForm(forms.Form):
     value = forms.CharField(max_length=1024, required=False)
-    data_url = forms.URLField(required=False)
```

Comment:
using a CharField instead of URL here because the data_url can either be an http url or an image blob

Reply:
When testing the tool icon editing, I noticed if I put a url for the icon then the post request has the url in the `value` field and if I select an image file for the icon, then the post request has the image data in `data_url` field.  So I am wondering should the 'value' form field be set to URLField. Anyway, this is not related to the original issue that this PR addresses.